src/features/messages/cards/TransactionCard.tsx

```
@@ -202,9 +208,25 @@ function TransactionDetails({
+  let formattedHash = hash;
+  switch (protocol) {
+    case ProtocolType.Radix:
+      formattedHash = hexToRadixCustomPrefix(hash, 'txid', metadata?.bech32Prefix);
+      break;
+    case ProtocolType.Cosmos:
+      formattedHash = strip0x(hash);
+      break;
+    default:
+  }
+
+  let formattedMailbox =
```

ESLint

[error] 'formattedMailbox' is never reassigned. Use 'const' instead.

(prefer-const)